### PR TITLE
Correct Harfbuzz public headers directory

### PR DESCRIPTION
**Pull request by tn0502**
cc @ebraminio

### CODE REVIEW

File: tools/ports/harfbuzz.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def process_dependencies(settings):
 def process_args(ports, args, settings, shared):
   if settings.USE_HARFBUZZ == 1:
     get(ports, settings, shared)
-    args += ['-Xclang', '-isystem' + os.path.join(ports.get_build_dir(), 'harfbuzz')]
+    args += ['-Xclang', '-isystem' + os.path.join(ports.get_dir(), 'harfbuzz', 'harfbuzz-' + TAG, 'src')]
```
Comment (kripken) [inline tools/ports/harfbuzz.py]: why is this change needed? we should have copied the project into the build dir, so the headers should be there. Perhaps we were just missing the `'src'` addition at the end?

Comment (tn0502) [reply tools/ports/harfbuzz.py]: Basically, this is what the folder structure looks like when Harfbuzz is extracted:
```
$ find .emscripten_ports -name hb.h
.emscripten_ports/harfbuzz/harfbuzz-1.7.5/src/hb.h
```
I'm not familiar with how ports works. I basically just looked at #6273, and @ebraminio asked me to take over :P  What would be the correct way to copy the headers over? Note that Harfbuzz doesn't have an `include` folder. Everything is in `src`.

Comment (tn0502) [reply tools/ports/harfbuzz.py]: I've added a `make install` step there. I hope it's the right to call `make()` twice. Looks a bit weird to me since I don't really know what it does...

File: tools/ports/harfbuzz.py [hunk 2]
```diff
@@ -18,11 +18,17 @@ def create():
...
+        '-H' + source_path,
+        '-DCMAKE_BUILD_TYPE=Release',
+        '-DCMAKE_INSTALL_PREFIX=' + dest_path,
         '-DFREETYPE_INCLUDE_DIRS=' + freetype_include_dirs,
-        '-DCMAKE_BUILD_TYPE=Release'])
-      shared.Building.make(['make', '-C' + dest_path])
+        '-DFREETYPE_LIBRARY=' + freetype_lib,
+        '-DHB_HAVE_FREETYPE=ON'
+      ])
+      shared.Building.make(['make', '-C' + dest_path, 'install'])
```
Comment (kripken) [inline tools/ports/harfbuzz.py]: does `make install` also do the normal `make` step of building the source files? lgtm if so.

Comment (tn0502) [reply tools/ports/harfbuzz.py]: Yes, it does. I originally had both `make` and `make install` here. Tested that it works after I deleted Harfbuzz from `.emscripten_cache` and `.emscripten_ports`.

Comment (kripken) [reply tools/ports/harfbuzz.py]: Great, thanks for verifying.

**Comment by ebraminio** [comment]
@tn0502 Thank you :)
